backend/src/Db.ts

```diff
@@ -779,6 +779,21 @@ class Db {
     return this.db.any(sql, queryParams)
   }
 
+  async getTransferTimes (sourceChainSlug: string, destinationChainSlug: string) {
+    const sql = `
+      SELECT bond_within_timestamp 
```

Comment:
suggestion: camelcase select value to access it in js

```suggestion
SELECT bond_within_timestamp AS "bondWithinTimestamp"
```